packages/fuel-indexer-tests/components/indices/fuel-indexer-test/fuel_indexer_test.yaml

```diff
@@ -2,7 +2,7 @@ namespace: fuel_indexer_test
 graphql_schema: packages/fuel-indexer-tests/components/indices/fuel-indexer-test/schema/fuel_indexer_test.graphql
 abi: packages/fuel-indexer-tests/contracts/fuel-indexer-test/out/debug/fuel-indexer-test-abi.json
 start_block: 1
-contract_id: fuel1dthcclayuwhce7t7jt8dlpvv2c0q4wsyj4xsudcek5gwc6nd5cnqj4qkua
+contract_id: ~
```

Comment:
- Remind me why we're removing this? 
- Keeping this tests the `#contract_conditional` in the codegen
- If we remove this and (for whatever reason) the `#contract_conditional` breaks, we won't see it

Reply:
> @deekerno
> 
>     * This change affects our examples too I believe? But I don't see testing examples in the testing steps (or any mention of the examples at all)?
> 
>     * Can you confirm that all examples work as they did previously ?

Ah, my apologies. Yes, the `hello-world` example works as expected. I'll add some testing steps.

As far as removing it, it was to get rid of the hard-coded contract strings everywhere. I suppose leaving it there is fine, as we don't (yet) have a way to programmatically compile a WASM blob to test out the contract conditional.

Reply:
@deekerno 
- Thanks, keep in mind that while we do want to use as few hard-coded strings as possible, keeping the contract_id in the manifest is actually how the indexer works 😄
- So I wouldn't consider putting the contract ID in the manifest as using magic strings
- (We will have to update it when the contract changes, but that's only annoying cause we're making changes (as devs) often. Users won't really experience that pain like we do)

Reply:
I guess too much optimization _can_ be a bad thing. 😅